### PR TITLE
Add a pull request template for PRs in rust-vmm

### PR DESCRIPTION
**Pull request by lauralt**
Make some of the most important requirements for a PR more obvious, by adding a pull request template file.

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Summary of the PR
+
+*Please summarize here why the changes in this PR are needed.*
+
+### Requirements
+
+Before submitting your PR, please make sure you addressed the following
+requirements:
+
+- All commits in this PR are signed (with `git commit -s`), and follow
```
Comment (andreeaflorescu) [inline pull_request_template.md]: Can you make these check boxes instead?

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  [the 50/72 git commit rule](https://www.midori-global.com/blog/2018/04/02/git-50-72-rule).
+- All added/changed functionality is tested.
```
Comment (andreeaflorescu) [inline pull_request_template.md]: I would rephrase this to: "All added/changed functionality has a corresponding unit/integration test" to be more explicit. Otherwise people can think this means only that you tested it locally.

File: pull_request_template.md (hunk 1 above)
Comment (lauralt) [reply pull_request_template.md]: done

**Review by andreeaflorescu — APPROVED** [review]

**Review by AlexandruCihodaru — APPROVED** [review]

**Review by jiangliu — APPROVED** [review]

**Review by andreeaflorescu — APPROVED** [review]
